src/index.js

```diff
@@ -304,3 +310,7 @@ module.exports = class Bitwap {
     this.engine.stop()
   }
 }
+
+module.exports = Bitswap
+
+function noop () {}
```

Comment:
didn't we agree that we would stop doing style refactorings that are based on personal preferences?